FreeAPS/Sources/Shortcuts/Carbs/carbPresetResult.swift

```diff
@@ -0,0 +1,42 @@
+import AppIntents
+import Foundation
+
+@available(iOS 16.0, *) struct carbPresetResult: AppEntity, Identifiable {
```

Comment:
Please capitalize this `struct` name and the file name to conform to how other `struct`s in the project are named to far, i.e. `carbPresetResult` -> `CarbPresetResult`

Reply:
@dnzxy Fixed

Reply:
File name is still lowercase.

Reply:
@dnzxy Xcode was misbehaving, fixed now